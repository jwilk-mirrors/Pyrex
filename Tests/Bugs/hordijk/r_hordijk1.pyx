cdef extern from "Python.h":
	ctypedef class types.ListType [object PyListObject]:
		pass

cdef class Spam(ListType):
	def __init__(self):
		raise StandardError("This is not a spanish inquisition")
