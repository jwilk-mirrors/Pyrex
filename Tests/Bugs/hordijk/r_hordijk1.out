Exception: This is not a spanish inquisition
